docs/v1/use-cases.md

```diff
@@ -32,6 +44,64 @@ Kubernetes cluster to publish their services using a service broker.  This
 would allow these operators to participate in the existing service broker
 ecosystem and grow their user base accordingly.
 
+### Search and Browsing Services
+
+#### Searching Services
+
+Consumers should be be able to search or filter their catalog by labels. For
+example, if I search for all services with 'catalog=database' the catalog
+will return the list of services that match that label. This assumes, of
+course, that producers are able to label their service offerings.
+
+#### Service Metadata
+
+Each service should have a list of metadata that it exposes in the catalog.
+If we're following the Cloud Foundry model you can view the list of metadata
+fields [here](https://docs.cloudfoundry.org/services/catalog-metadata.html).
+
+We should consider what metadata needs to be exposed for a strong CLI and UI
+experience. Here are some suggestions for metadata fields:
+
+    * name
+    * short description
+    * long desciption
+    * documentation/support urls
+    * icon URL
+    * image URLs - a list of images that could be displayed in a UI
+    * TOS link
+    * a list of plans
+        * plan name
+        * plan description
+        * plan cost
+    * construction parameters
+        * name
+        * description
+        * default value
```

Comment:
Based on the above item 3 under "Searching and Browsing Services", it seems like there should also be an "optional" parameter